.travis.yml

```diff
@@ -0,0 +1,41 @@
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2017.
+# Part of the SW360 Portal Project.
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# initial author: maximilian.huber@tngtech.com
+# -----------------------------------------------------------------------------
+# dist: precise
+dist: trusty
+sudo: required
+
+services:
+  - docker
+
+install:
+  - curl -L https://github.com/docker/compose/releases/download/1.14.0/docker-compose-`uname -s`-`uname -m` | sudo tee /usr/local/bin/docker-compose > /dev/null
+  - sudo chmod +x /usr/local/bin/docker-compose
+  
+env: CONFIGURATION_ENV=""
+
+script:
+  - set -e
+  - cd deployment
+  - echo "$CONFIGURATION_ENV" >> configuration.env
+  - ./docker-compose.sh --prepare
+  - ./docker-compose.sh build
+  - ./docker-compose.sh up -d
+  - sleep 120
+  - ./docker-compose.sh logs sw360 | grep -iq "Determine dialect for PostgreSQL 9"
+  - ./docker-compose.sh logs sw360couchdb | grep -iq "Apache CouchDB has started"
+  - curl -L -S -k https://localhost:8443 | grep -iq "html"
```

Comment:
This test is not sufficient. e.g. the bad gateway page from ngnix also includes "html".